mlkaps/MLKaps.py

```diff
@@ -50,6 +50,7 @@ def run_clustering(args, experiment_config, optim_results):
     # Properly re-implement it
     clustered_models = generate_clustering_models(experiment_config, optim_results)
 
+    # from mlkaps.clustering import plot_all_decision_tree, plot_all_decisions_maps
```

Comment:
even if it does not hurt, we should probably avoid leaving commented debug statement.

Reply:
There is a lot of cleanup that could be done. Maybe this should be done in a separate PR?